src/v3/src/transformer/terminal/transformTerminalMessages.ts

```diff
@@ -75,10 +75,6 @@ const appendBiometricsErrorBox = (
     options: {
       class: 'ERROR',
       dataSe: 'callout',
-      listOptions: {
-        type: 'ul',
-        items: listMessages,
-      },
```

Comment:
Remnant from the WidgetMessage refactor. The infoBoxElement no longer respects the listOptions prop since it is now apart of the WidgetMessage.